plugins/by-name/nui/default.nix

```diff
@@ -0,0 +1,9 @@
+{ lib, ... }:
+lib.nixvim.neovim-plugin.mkNeovimPlugin {
+  name = "nui";
+  originalName = "nui.nvim";
+  package = "nuinvim";
```

Comment:
I think the package `nui-nvim`.
I also think that the plugin has no setup function, you'll need to disable it

Reply:
Sry, wasn't meant to be ready for review ^^. I haven't tested it yet. But yeah, you're right, syntax error and correct the setup